src/_data/languages_base.json

```diff
@@ -48,7 +49,8 @@
     "rssFeedHeading": "Feed RSS",
     "rssFeedParagraph": "Suscríbete a las últimas noticias de OWA.",
     "siteFooterPoweredBy": "Este sitio funciona con Netlify",
-    "skipLinkText": "Saltar al contenido"
+    "skipLinkText": "Saltar al contenido",
+    "translationNotAvailable": "Lamentablemente, esta página aún no se ha traducido a su idioma preferido. Si desea ayudarnos a traducirlo, únase a #translation en OWA Discord."
```

Comment:
@nickchomey @AlfonsoML could you please confirm if this translation to spanish needs any edits?

Reply:
@AlfonsoML did you have any thoughts to add here?

Reply:
Sorry for the delay.
I would use a mix of both, using "tu", proper genre and "Discord de OWA"

Lamentablemente, esta página aún no se ha traducido a tu idioma preferido. Si deseas ayudarnos a traducirla, únete a #translation en el Discord de OWA .

Reply:
Thanks, I'll update!